ports/nrf/boards/ARDUINO_NANO_33_BLE_SENSE/mpconfigboard.h

```diff
@@ -13,6 +13,8 @@
 
 #define MICROPY_PY_MACHINE_UART         (1)
 #define MICROPY_PY_MACHINE_HW_PWM       (1)
+#define MICROPY_PY_MACHINE_HW_SPI       (1)
+#define MICROPY_PY_MACHINE_TIMER        (1)
```

Comment:
I don't think this will enable `machine.Timer`, you need to still add the correct entry to the `machine` globals table.  Maybe you meant to use `MICROPY_PY_MACHINE_TIMER_NRF`?

Reply:
Yes I did, thanks! I think it was renamed at some point.